airbyte-cdk/bulk/core/load/src/integrationTest/kotlin/io/airbyte/cdk/mock_integration_test/MockDestinationCheck.kt

```diff
@@ -0,0 +1,9 @@
+package io.airbyte.cdk.mock_integration_test
+
+import io.airbyte.cdk.check.DestinationChecker
+import javax.inject.Singleton
+
+@Singleton
+class MockDestinationCheck: DestinationChecker<MockDestinationConfiguration> {
```

Comment:
I'm adding this stuff to integrationTest to avoid weird accidental micronaut injections (in both directions - I don't want to accidentally break a unit test by declaring a singleton here. And, at least for now, there's not a great way to call CliRunner with arbitrary additional envs, so we can't opt these tests into specific mocks)